core_backend/app/dashboard/models.py

```diff
@@ -787,6 +787,7 @@ async def get_ai_answer_summary(
             ContentFeedbackDB.feedback_datetime_utc >= start_date,
             ContentFeedbackDB.feedback_datetime_utc < end_date,
             ContentFeedbackDB.feedback_text.is_not(None),
+            ContentFeedbackDB.feedback_text != "",
```

Comment:
works well for empty strings, though if I put blank spaces it still gets counted... Can we check for that also while we're at it? Like run "strip" somewhere? Or change empty or whitespace text into null when inserting into the DB in the first place